lib/urlwatch/command.py

```diff
@@ -142,6 +142,21 @@ def test_filter(self, id):
         # (ignore_cached) and we do not want to store the newly-retrieved data yet (filter testing)
         return 0
 
+    def run_new_jobs(self):
+        new_jobs = list()
+        for idx, job in enumerate(self.urlwatcher.jobs):
+            history_data = self.urlwatcher.cache_storage.get_history_data(job.get_guid(), 1)
+            if not history_data:
+                logger.info('Add Job: ' + job.pretty_name())
+                new_jobs.append(idx+1)
+
+        if len(new_jobs) != 0:
```

Comment:
This can be simplified as:

```suggestion
        if new_jobs:
```